src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -51,6 +51,11 @@ public static void loadJni() {
     return;
   }
 
+  // temp, for kotlin_library
+  public long getDeviceControllerPtr() {
```

Comment:
please file a ticket for follow-up on this, thanks

Reply:
Add #32133.